install_config/web_console_customization.adoc

```diff
@@ -256,6 +256,7 @@ assetConfig:
   extensions:
     - name: about
       sourceDirectory: /path/to/about
+  hawtioPluginLoader.addModule('aboutPageExtension');
```

Comment:
this needs to go above with the code for the angular module, this is content within the about.js file